website/docs/quick-start-guide.md

```diff
@@ -1028,6 +1028,13 @@ Notice that the save mode is `Append`.
 </TabItem>
 <TabItem value="python">
 
+#### Code overview 
+Soft deletes retain the record key and null out the values for all the other fields. For example, records with nulls in soft deletes are always persisted in storage and never removed.<br/><br/>
```

Comment:
This seems like redundant info already available in the lines 962 - 965 above. Do we need this specifically for Python tab?

Reply:
This is just like a friendly reminder about the code below, so user doesn't have to scroll up and refresh

Reply:
Makes sense. Nadine and I synced on this offline. We will pull this out of individual tabs and add a brief description under hard deletes section for better readability